src/Microsoft.TemplateEngine.Cli/TemplateListResolutionResult.cs

```diff
@@ -258,6 +267,16 @@ public bool UsingContextMatches
             }
         }
 
+        // If BaseGetBestTemplateMatchList returned only partial matches with template name/lang, this is true.
```

Comment:
please use the standard comment notation
```///``` triggers built in schema of .NET comments.

So far, I've been recommending we comment...

1. All public members
1. non-public members that require explanation

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments